mowgli_etl/pipeline/wdc/wdc_size_buckets.py

```diff
@@ -0,0 +1,149 @@
+from typing import Tuple
+from mowgli_etl.pipeline.wdc.wdc_product_type import WdcProductType
+from mowgli_etl.pipeline.wdc.wdc_product_dimensions import WdcProductDimensions
+from mowgli_etl.pipeline.wdc.wdc_generic_size import WdcGenericSize
+
+
+class WdcSizeBuckets:
+    def __init__(self):
+        self.averages = dict()
+
+    def __bucket(self, dimension):
+        # Determine heuristics for buckets
+        return None
+
+    def __convert(self, dimension, origin):
+        if origin in (
+            "in",
+            "lb",
+            "lbs",
+            "v",
+        ):
+            return dimension
+        if origin == "mm":
+            return dimension * 3.9
+        if origin == "cm":
+            return dimension * 0.39
+        if origin == "m":
+            return dimension * 0.0039
+        if origin == "ft":
+            return dimension * 12
+        if origin == "mv":
+            return dimension / 1000
+        if origin == "kv":
+            return dimension * 1000
+        if origin == "oz":
+            return dimension / 16
+        if origin == "g":
+            return dimension * 0.0022046
+        if origin == "mg":
+            return dimension * 0.0000022046
+        if origin == "kg":
+            return dimension * 2.2046
+
+    def __update(self, generic):
```

Comment:
Please add type annotations throughout here, `generic` doesn't tell me what it really is.